test/parallel/test-tls-keylog-tlsv13.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+if (/^(0\.|1\.0\.|1\.0\.0)/.test(process.versions.openssl))
```

Comment:
```suggestion
if (/^(0\.|1\.0\.|1\.1\.0)/.test(process.versions.openssl))
```

^--- typo in third part of regex

Reply:
I applied the change to unblock this, and I'll kick off ci.